src/sentry/api/endpoints/artifact_bundles.py

```diff
@@ -12,8 +14,21 @@
 from sentry.utils.db import atomic_transaction
 
 
+class ArtifactBundlesMixin:
+    @classmethod
+    def derive_order_by(cls, sort_by: str) -> Optional[str]:
+        is_desc = sort_by.startswith("-")
+        sort_by = sort_by.strip("-")
+
+        if sort_by == "date_added":
+            order_by = "date_uploaded"
+            return f"-{order_by}" if is_desc else order_by
+
+        return None
```

Comment:
it's totally fine to raise exception here
>            return Response(
                {"error": "You can either sort via 'date_added' or '-date_added'"}, status=400
            )
instead of return None and then again check if not None.